fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -251,4 +229,11 @@ public void getUserAuthorization(@PathVariable String userId,
 
     return Optional.ofNullable(userPermission);
   }
+
+  private UserPermission.View getUserPermissionView(String userId) {
+    return getUserPermissionOrDefault(userId)
+        .orElseThrow(NotFoundException::new)
+        .getView()
+        .setAllowAccessToUnknownApplications(configProps.isAllowAccessToUnknownApplications());
```

Comment:
Opt'd to just tack it on to the `UserPermission.View` rather than actually store it as an attribute of the `UserPermission` itself. 

Seemed alright?